Application/src/view/visuals/Draw.java

```diff
@@ -303,6 +327,32 @@ void renderClassDiagramClass(){
         }
     }
 
+    /**
+     * Places the relationships in the class diagram
+     */
+    void renderClassRelationship(){
+        if (allClassRelationships.size()==0) return;
+        int space = ((int)this.canvas_class.getWidth())/this.allClassRelationships.size();
+
+        // Drawing 3 mocked up relationships
+
+        // Place first relationship
```

Comment:
No hardcoded stuff plz

Reply:
@timjon "TODOs later on: - integration with decoded parsed data." ;)

Reply:
Doing the integration later is fine, what im not fine with is that its hardcoded so we cant accually test if it works for any other cases

Reply:
@timjon alright thanks for the clarification, in order to test you have to do the following

in draw.java

first you initialise the relationship in the Draw() method

ClassRelationship cl= new ClassRelationship(allClassDiagramClasses.get(0).getCoordinates(),
                allClassDiagramClasses.get(1).getCoordinates(), 40);
        allClassRelationships.add(cl);
replace get(0) with get(YOURINDEX) with a index of your choice to choose the super class
then replace the other get(1) with get(YOURINDEX) with a index of your choice to choose the sub class

then in the renderClassRelationship() change the indexes accordingly 
Coordinates fromNode = allClassDiagramClasses.get(YOURINDEX).getCoordinates();
Coordinates toNode = allClassDiagramClasses.get(YOURINDEX).getCoordinates();

leave this at index 0 if you only have 1 relationship
allClassRelationships.get(0).init(fromNode,toNode,(space/2));

Reply:
Im fine with you having the first section since its mock data, what im refering to with the hardcoded part is this:
`then in the renderClassRelationship() change the indexes accordingly
Coordinates fromNode = allClassDiagramClasses.get(YOURINDEX).getCoordinates();
Coordinates toNode = allClassDiagramClasses.get(YOURINDEX).getCoordinates();

leave this at index 0 if you only have 1 relationship
allClassRelationships.get(0).init(fromNode,toNode,(space/2));`

This should be made dynamic so that a relationship gets drawn up automatically if added to the list of relationships

Reply:
@timjon you are right of course, this isn't dynamic at all if we leave it like that. i am changing it in the integration part, but keeping it now for testing/previewing the mocked part

Reply:
And the thing im saying is that I think it should be fixed before that since its related to the integration with the old code.